### DISCUSSION TITLE
Need help exiting git:(main) in os x terminal

### DISCUSSION THREAD

**Discussion by saraschoonover** [nss-evening-cohort-14/evening-client-side#11] [Help Me!] [answered]
I'm currently working on another repo and whenever I now try go to my OS X terminal before I switch to vscode and Github and after I've gotten into the exercises file, I now see: exercises git:(main) 

What have you googled?
https://github.community/t/need-help-exiting-git/335
https://stackoverflow.com/questions/61491408/how-to-exit-from-gitmaster

What have you tried? Tell us everything you have tried.
I've tried :qw 
cd

**Comment by ajawashington** [comment]
Please use this format #2 when submitting issues.

**Comment by ajawashington** [comment, marked as answer]
So I'm not really sure what your issue is, but git isn't something you can quit out of. If you have initialized git in a directory it will always show you the branch (i.e. git:(main) you are on. **Do not remove the git file.** Let me know if this helps you understand that a little better.
